mathics/core/expression.py

```diff
@@ -1937,3 +1924,14 @@ def convert_expression_elements(
 
 def string_list(head, elements, evaluation):
     return atom_list_constructor(evaluation, head, "String")(elements)
+
+
+# These are constant expressions used in several places.
+# TODO: See if the is a better place to declare them.
```

Comment:
There is no reason this can't be put in its own separate module as was done between the symbols and systemsymbols module .

More smaller focused modules is better than one sprawling module

Reply:
OK, any suggestions for the name of such a module?

Reply:
These all seem to be symbolic constants. 

So `mathics.core.symboli_constants`.

Reply:
At one time I had thought that we might set elements_properties, but that is not needed - once set on the first use will no longer need to get set again. 

(And computing the properties for any of these is very quick.)

We might consider if it is worth it to turn this create an ExpressionConstant class which has a special and simple evaluate method (and other methods like that).

Reply:
OK, maybe I can make a separate PR with these objects and module, and then reformulate over it.

Reply:
I think this was solved in #831